tajo-plan/src/main/java/org/apache/tajo/plan/util/PlannerUtil.java

```diff
@@ -133,11 +134,90 @@ public static boolean checkIfSimpleQuery(LogicalPlan plan) {
           }
         }
       }
+
+      if (!noWhere && scanNode.getTableDesc().isExternal() && scanNode.getTableDesc().getPartitionMethod() != null) {
```

Comment:
A managed table can also be partitioned. 
Would you explain why you added the isExternal() condition?